locales/nb/validation-inline.php

```diff
@@ -1,131 +1,143 @@
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => ':attribute må aksepteres.',
+    'active_url'           => ':attribute er ikke en gyldig URL.',
+    'after'                => ':attribute må være en dato etter :date.',
+    'after_or_equal'       => ':attribute må være en dato etter eller lik :date.',
+    'alpha'                => ':attribute må kun bestå av bokstaver.',
+    'alpha_dash'           => ':attribute må kun bestå av bokstaver, tall og bindestreker.',
+    'alpha_num'            => ':attribute må kun bestå av bokstaver og tall.',
+    'array'                => ':attribute må være en matrise.',
+    'attached'             => ':attribute er allerede vedlagt.',
+    'before'               => ':attribute må være en dato før :date.',
+    'before_or_equal'      => ':attribute må være en dato før eller lik :date.',
     'between'              => [
-        'array'   => 'This content must have between :min and :max items.',
-        'file'    => 'This file must be between :min and :max kilobytes.',
-        'numeric' => 'This value must be between :min and :max.',
-        'string'  => 'This string must be between :min and :max characters.',
+        'numeric' => ':attribute må være mellom :min - :max.',
+        'file'    => ':attribute må være mellom :min - :max kilobytes.',
+        'string'  => ':attribute må være mellom :min - :max tegn.',
+        'array'   => ':attribute må ha mellom :min - :max elementer.',
     ],
-    'boolean'              => 'This field must be true or false.',
-    'confirmed'            => 'The confirmation does not match.',
-    'date'                 => 'This is not a valid date.',
-    'date_equals'          => 'This must be a date equal to :date.',
-    'date_format'          => 'This does not match the format :format.',
-    'different'            => 'This value must be different from :other.',
-    'digits'               => 'This must be :digits digits.',
-    'digits_between'       => 'This must be between :min and :max digits.',
-    'dimensions'           => 'This image has invalid dimensions.',
-    'distinct'             => 'This field has a duplicate value.',
-    'email'                => 'This must be a valid email address.',
-    'ends_with'            => 'This must end with one of the following: :values.',
-    'exists'               => 'The selected value is invalid.',
-    'file'                 => 'The content must be a file.',
-    'filled'               => 'This field must have a value.',
+    'boolean'              => ':attribute må være sann eller usann',
```

Comment:
In validation-inline.php we don't have :attribute

Reply:
Will fix.